earth_enterprise/src/fusion/dbmanifest/dbmanifest.cpp

```diff
@@ -262,6 +263,7 @@ void DbManifest::GetManifest(geFilePool &file_pool,
     // assetroot).
     geindex::IndexManifest::GetManifest(file_pool, fusion_config_.index_path_,
                                         *stream_manifest, tmp_dir, "", "");
+    notify(NFY_DEBUG, "Processing %ld POI files with empty prefix", fusion_config_.poi_file_paths_.size());
```

Comment:
Are these new debug statements intended to be permanent?

Reply:
Some I can remove if we think they are excessive.  But most of them did prove helpful and could be helpful later.  I *think* this one would be good to keep (but I could be persuaded otherwise).  But let me know if there are ones you think are excessive.  All the new tracing I added where debug level or  verbose level.  These levels don't log by default.

Reply:
They could also be commented out for now...though in theory they are already tagged as debug and so I would expect them to be runtime disableable.

Reply:
Yes by default all these messages are disabled at run-time.  You have to start fusion with an environment variable (KH_NFY_LEVEL) set to a numeric value of 6 (for NFY_DEBUG) or 7 (for NFY_VERBOSE).  Otherwise the default is info (NFY_INFO/4).

Reply:
I didn't realize they were all DEBUG or VERBOSE statements. Given that, I think they're fine.